etl-pipeline/scripts/addHasPartToGRINRecords.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import boto3
+from sqlalchemy import func, or_
+import file_conversion.pdfs.mets_parser as mets_parser
+
+
+from logger import create_log
+from digital_assets.utils.get_stored_file_url import get_stored_file_url
+from model import Record, Part, Source, FileFlags, GRINStatus, GRINState
+from managers import DBManager, SQSManager
+
+
+s3_client = boto3.client("s3")
```

Comment:
I think it's best practice to instantiate this in main and then pass it to `get_mets_file_from_s3`